custom_components/tuya_local/devices/hombli_thermostatic_radiator_valve.yaml

```diff
@@ -0,0 +1,58 @@
+name: Hombli Thermostatic Radiator Valve
+products:
+  - id: ngxyxyc8
+    name: Hombli HBRT-0109
+primary_entity:
+  entity: climate
+  dps:
+    - id: 8
+      type: boolean
+      name: window_check
+      icon: "mdi:window-open"
```

Comment:
Don't set the icon at dp level without a condition, it is no different than setting it at entity level, but causes confusion because another unfamiliar user will think it is related to the dp.  If window_check is not used in the climate entity, it should be its own binary_sensor or switch entity.